developer-docs-site/docs/reference/telemetry.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Telemetry"
+slug: "telemetry"
+---
+
+At Aptos Labs, we develop software and services for the greater Aptos community and ecosystem. On top of community feedback, we use telemetry to help better understand how our software is being deployed in the ecosystem.
+
+The Aptos node binary collects telemetry such as software version, operating system information, and IP address. See [Types of information collected](#types-of-information-collected).
+
+The Aptos node binary does **not** collect personal information such as usernames or email addresses.
+
+Users can disable telemetry at any point. If telemetry remains enabled, Aptos node binary will send telemetry data in the background.
+
+# Disabling telemetry
+
+On macOs and Linux, you can disable telemetry by setting the `APTOS_TELEMETRY_OPTOUT` environment variable to any value.
+
+```
+export APTOS_TELEMETRY_OPTOUT=true
```

Comment:
OPTOUT looks like OUTPUT, which makes this env var a bit confusing at first look. maybe use something like DISABLE?

Reply:
good point. renamed